src/main/java/io/stargate/sgv2/jsonapi/service/resolver/CommandResolverService.java

```diff
@@ -40,7 +41,7 @@ public <T extends Command> Uni<CommandResolver<T>> resolverForCommand(T command)
     // try to get from the map of resolvers
     return Uni.createFrom()
         .item((CommandResolver<T>) resolvers.get(command.getClass()))
-
+        .runSubscriptionOn(Infrastructure.getDefaultWorkerPool())
```

Comment:
Could use a comment on purpose of this? (sounds like it's async operation for some reason?)